cypress/support/step_definitions/dynamicCalls.js

```diff
@@ -51,22 +51,40 @@ Given(
 
 /**
  * @module getterSetterCalls
- * @function Given 1st party app invokes the '(.+)' API to set( invalid)? value
+ * @function Given 1st party app(?: invokes the '(.+)' API)? to set(?: '(.*?)' to( invalid)? '(.*?)'|( invalid)? value)?
  * @description Sending a message to platform to set a value and `invalid` is a optional parameter that specify whether to expect for an error or a result.
  * @param {String} sdk - sdk name.
+ * @param {String} attribute - attribute holds the value of the method name
  * @param {String} invalidValue - Determines whether expecting for an error or result.
+ * @param {String} value - value holds the value used for to set the value or for validation.
+ * @param {String} invalidValue1 - Determines whether expecting for an error or result.
  * @example
- * Given 1st party app invokes the 'Firebolt' API to set
+ * Given 1st party app invokes the 'Firebolt' API to set value
+ * Given 1st party app to set value
  * Given 1st party app invokes the 'Firebolt' API to set invalid value
+ * Given 1st party app invokes the 'Firebolt' API to set 'enabled' to 'true'
+ * Given 1st party app to set 'enabled' to 'true'
```

Comment:
Some of this will be removed once we migrate manage tests? If yes could we maybe mark which examples are deprecated?

Reply:
Added a note, is this fine
`Note: The below formats will be deprecated in the future`.